### ISSUE TITLE
Unusual _ for 'from' parameter

### ISSUE THREAD

**Issue by deios0** [LautaroParada/eod-data#41]
Hello, you use 'from_' in your documentation for some functions, but I believe it should be 'from':

resp = client.get_prices_eod('AAL.LSE', period='d', order='a', from_='2017-01-05')

Not only this one, there are more of them. Thank you.

**Comment by LautaroParada** [comment]
Hi @deios0, thank you for you submitting your feedback. I have chosen the `from_` syntaxis rather than the `from` because Python doesn't permit using the former as a valid argument or parameter for a method (At least Anaconda doesn't let you). So to address that issue, I have created a special class that transforms the `from_` into `from` for a valid API request. You can find the details of that transformation inside the [request_handler_class](https://github.com/LautaroParada/eod-data/tree/main/eod/request_handler_class).

**In short:** The underscore is intentional, so you don't have to worry. Just stick to the documentation.
